README.md

```diff
@@ -333,7 +333,7 @@ print(rules.lookup(typ=MyAmbiguousClass, verb='show_pattern'))
 
 _The RuleSet caches encoders._ Construct a new ruleset if you want to change settings.
 
-_Encoders and decoders do very little checking._ Especially, if you're translating
+_Encoders and decoders do a very little checking._ Especially, if you're translating
```

Comment:
I don't believe this change is correct.

```suggestion
_Encoders and decoders do very little checking._ Especially, if you're translating
```

Reply:
"very little checking" is a subject, AFAIK. And it is countable: checking/checkings ("checkings" is clumsy but correct).

Reply:
Sorry I didn't see this earlier; at least it's not too late for a grammar lesson. English is a subject-verb-object language, so anything after the verb is going to be an object, not a subject. "Checking" is a gerund, and gerunds are generally mass nouns, so the article "a" is not correct. You can count some number of checks, but the process of checking is not countable.